src/kirby/components/modal/modal.component.ts

```diff
@@ -34,7 +34,8 @@ export class ModalComponent implements AfterViewInit {
   onModalDismiss(e: any) {
     // Handle key press, due to:
     // https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/button_role#Required_JavaScript_Features
-    if (e && e.keyCode !== 32 && e.keyCode !== 13) {
+    if (e && e.keyCode && e.keyCode !== 32 && e.keyCode !== 13) {
+      console.log(`interrupting dismiss, keycode: ${e.keyCode}`);
```

Comment:
I'm thinking this console.log should probably not make it to master? Otherwise no comments. :)